app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -25,8 +25,13 @@ if [ -f /etc/os-release ] ; then
   . /etc/os-release
 fi
 
+SSL_CA_CERT=$(mktemp --suffix .crt)
+tee $SSL_CA_CERT << CA
```

Comment:
I wonder why you use tee instead of `cat`. Tee also outputs it to stdout but you don't really need that, right?